Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.py

```diff
@@ -0,0 +1,602 @@
+            url = f'{object_type}/{object_type_id}/drive/items/{item_id}/content'
+
+        # send request
+        url = BASE_URL + f'/{url}'
+        with open(file_path, 'rb') as file:
+            self.headers['Content-Type'] = 'application/octet-stream'
+            return self.http_call('PUT', full_url=url, data=file, headers=self.headers, url_suffix='')
+
+    def delete_file(self, object_type, object_type_id, item_id):
+        if object_type == 'drives':
+            url = f'{object_type}/{object_type_id}/items/{item_id}'
+
+        elif object_type in ['groups', 'sites', 'users']:
+            url = f'{object_type}/{object_type_id}/drive/items/{item_id}'
+
+        # send request
+        url = BASE_URL + f'/{url}'
+        self.headers['Content-Type'] = 'text/plain'  # request returned empty and can not decoded to json
+        if '' == self.http_call('DELETE', full_url=url, headers=self.headers, url_suffix='', resp_type='text'):
+            # resp_type='text' returned empty and can not decoded to json
+            return f'Item was deleted successfully'
+
+    def upload_new_file(self, object_type, object_type_id, parent_id, file_name, entry_id):
+        """
+        this function upload new file to a selected folder(parent_id)
+        :param object_type: drive/ group/ me/ site/ users
+        :param object_type_id: the selected object type id.
+        :param parent_id:
+        :param file_name:
+        :param entry_id:
+        :return:
+        """
+        file_path = demisto.getFilePath(entry_id).get('path')
+
+        if 'drives' == object_type:
+            url = f'{object_type}/{object_type_id}/items/{parent_id}:/{file_name}:/content'
+
+        elif object_type in ['groups', 'users', 'sites']:
+            url = f'{object_type}/{object_type_id}/drive/items/{parent_id}:/{file_name}:/content'
+            # for sites, groups, users
+        url = BASE_URL + f'/{url}'
+        with open(file_path, 'rb') as file:
+            self.headers['Content-Type'] = 'application/octet-stream'
+            return self.http_call('PUT', full_url=url, headers=self.headers, url_suffix='', data=file)
+
+    def download_file(self, object_type, object_type_id, item_id):
+        if object_type == 'drives':
+            url = f'{object_type}/{object_type_id}/items/{item_id}/content'
+
+        elif object_type in ['groups', 'sites', 'users']:
+            url = f'{object_type}/{object_type_id}/drive/items/{item_id}/content'
+
+        # send request
+        url = BASE_URL + f'/{url}'
+        res = self.http_call('GET', full_url=url, headers=self.headers, url_suffix='', resp_type='')  # it needs
+        # resp_type='' to stay empty because if response type is empty http_requests returned the raw response.
+        # I need it because graph api returnes an response as text without content so res.text fails
+        return res
+
+    def create_new_folder(self, object_type, object_type_id, parent_id, folder_name):
```

Comment:
this function needs documentation. see `get_encrypted()`(in your code) for example.